docs/publications/BioHackrXiv/paper.md

```diff
@@ -0,0 +1,154 @@
+
+Figure \ref{annotation_workflow} shows a step-by-step breakdown of the above process.
+
+![Step-by-step workflow for systematically improving metadata annotations across bio.tools registry entries and Galaxy tool wrappers. After selecting a Galaxy tool and checking for the presence of a bio.tools ID in its XML file, a curator needs to review bio.tools, create a new bio.tools entry (if needed), and then ensure that both this entry and the Galaxy tool XML file are up-to-date. Updating bio.tools makes use of the registry wizard, and updating a Galaxy tool wrapper to include a bio.tools xref snippet requires a pull request against the development repository. \label{annotation_workflow}](./annotation.png)
+
+
+# Outcomes and results
+
+There were multiple concrete outcomes from this BioHackathon project, including the interactive Galaxy tools table, a process for updating bio.tools, an in development GTN tutorial describing this process, and an update to the Galaxy IUC standards for tool wrapping. These are described in more detail below.
+
+
+## Prototype interactive table for Galaxy communities
+
+The described workflow for the Galaxy tool metadata extractor (see Figure \ref{metadata_extractor_pipeline}) was successfully implemented (cite GitHub) and could collect 1,294 Galaxy tool suites (cite output). Of those tool suites, only 267 had a bio.tools identifier, which highlights the importance of performing the annotation process in parallel and complementing the tools with additional metadata. An example view of the created interactive table is shown in Figure \ref{web_table}. The filtered table for the microGalaxy community is embedded in the microGalaxy Galaxy Hub (cite) page as well as the dedicated microGalaxy subdomain (cite). 
+
+![Screenshot of the interactive web table. The table provides comprehensive metadata for all Galaxy wrappers of a specific community and allows for custom searches based on logic filters over all columns. In the shown example, the user queries for all up-to-date tools that are annotated with the EDAM Operation assembly. \label{web_table}](./figures/itable.png)
+
+
+## bio.tools and EDAM annotations for microbiome community
+
+During the week of the Biohackathon the microGalaxy community executed the annotation workflow as described in the Methods Section (see also Figure \ref{annotation_workflow}). The initial filtered tool table of the microGalaxy included 218 tool suites, of which 61 had corresponding bio.tools identifiers. The process of the work was tracked using a GitHub project board. After the annotation process, the number of tools with bio.tools annotations was increased to 107. The added annotations for each respective bio.tools entry were also collectively reviewed by the team. A rerun of the Galaxy tool metadata extractor pipeline collected the additional information, and the metadata is now included in the interactive microGalaxy tool table. 
```

Comment:
```suggestion
During the week of the Biohackathon, the microGalaxy community executed the annotation workflow as described in the Methods section (see also Figure \ref{annotation_workflow}). The initial filtered tool table of the microGalaxy included 218 tool suites, of which 61 had corresponding bio.tools identifiers. The process of the work was tracked using a GitHub project board. After the annotation process, the number of tools with bio.tools annotations was increased to 107. The added annotations for each respective bio.tools entry were also collectively reviewed by the team. A rerun of the Galaxy tool metadata extractor pipeline collected the additional information, and the metadata is now included in the interactive microGalaxy tool table. 
```

Reply:
Do the numbers 61 and 107 anyhow correspond to the "> 50" mentioned higher above? I'm confused.

Reply:
At the start of the work 61 tools had bio.tools entries
When finished, this number had increased to 107
The number in the introduction could be update to reflect this better?